### DISCUSSION TITLE
code

### DISCUSSION THREAD

**Discussion by maali69** [eritislami/evobot#578] [Q&A]
I would like to know the code that auto delete the bot's messages. e.g "the music player auto deletes its self after the song ends".

**Comment by eritislami** [comment]
You didn't explain it very well but i think you are looking for this
https://github.com/eritislami/evobot/blob/ac378df48f7e5420306693c01a1b3712662ac11e/include/play.js#L190-L195
It checks if pruning is true or false from config.json and if it is true it will delete the last "playingMessage" which is a message that the bot sends when it starts playing a video. Hope this was what you are looking for.

### Also would like to remind you that this is not a place to ask for programming support, if you need help with javascript or discord.js please google it or check relevant resources to learn.
